packages/OreIdWalletConnect/src/ConnectionsBadge/ConnectionsBadge.tsx

```diff
@@ -16,9 +16,9 @@ export const ConnectionsBadge: React.FC<ConnectedUIProps> = ({ isListening, peer
   return (
     <div className="oreIdWalletConnect-connectionsBadge">
       <div className="oreIdWalletConnect-connectionsBadge-icons">
-        {peerMeta.map((metaData) => {
+        {peerMeta.map((metaData, index) => {
           if (!metaData?.icons?.[0]) return null
-          return <ConnectionIcon key={metaData.url} icon={metaData.icons[0]} size={20} />
```

Comment:
Don't use the array index with key if you have another option, it generates bugs... If you delete an item in the middle of the array you will have inconsistency problems to render.
only use index if you don't have another more suitable field or if you're sure what you're doing.

Reply:
there is no suitable field currently